airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx

```diff
@@ -170,6 +172,9 @@ export const AuthenticationProvider: React.FC<React.PropsWithChildren<unknown>>
       hasPasswordLogin(): boolean {
         return !!state.providers?.includes("password");
       },
+      hasCorporateEmail(): boolean {
```

Comment:
Added this new function here so it can be used across the whole app because I feel we will start segmenting more and more experiments only for corporate emails.
Let me know if this is not the right place or there’s a better one

Reply:
I think that might be a valid place for it. Alternatively we would put it into `src/utils` into a utility function, but I think this can live on the `AuthService` as well.